src/main/webapp/app/course/manage/detail/course-detail-doughnut-chart.component.html

```diff
@@ -1,5 +1,5 @@
 <div class="doughnut mx-3">
-    <a *ngIf="titleLink" [routerLink]="['/course-management', courseId, titleLink]">
+    <a *ngIf="titleLink" [routerLink]="['/course-management', course.id, titleLink]">
```

Comment:
I think using `course.isAtLeastInstructor` condition is better rather than checking whether the `titleLink` exists.
As done here: https://github.com/ls1intum/Artemis/blob/develop/src/main/webapp/app/exercises/shared/exam-exercise-row-buttons/exam-exercise-row-buttons.component.html#L3
What do you think?

Reply:
hmm, using he condition in there would affect the other clickable chart titles for tutors and editors in the course.